source/MLKit/Core/Enums.cs

```diff
@@ -4,8 +4,15 @@
 namespace MLKit.Core {
 	[Native]
 	public enum ImageSourceType : long {
-		ImageSourceTypeImage = 0,
-		ImageSourceTypePixelBuffer = 1,
-		ImageSourceTypeSampleBuffer = 2
+		Image = 0,
+		PixelBuffer = 1,
+		SampleBuffer = 2
+	}
+
+	[Native]
+	public enum ModelDownloadUserInfoKey : long
```

Comment:
I don't understand this. It doesn't look like it should be an enum. It's a constant NSString, and so should be defined as such in ApiDefinition.cs, right?

I'm assuming you added it for a reason though. Can you add some context?

Reply:
Same as above, missed this one in my cleanup

Reply:
alright, I think I've updated this to match the pattern I've seen in other libraries in the repo and it should accurately reflect the underlying header files.